src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -12,22 +12,26 @@
 public class EmailScheduler {
 
     private static final String SUBJECT = "Tasks: Once a day email";
+
     @Autowired
     private SimpleEmailService simpleEmailService;
 
     @Autowired
     private TaskRepository taskRepository;
 
+
     @Autowired
     private AdminConfig adminConfig;
 
-    @Scheduled(cron = "0 0 10 ***")
+    private String createMessage() {
+        return "Currently in database you got: " + taskRepository.count() + (taskRepository.count() == 1 ? "task" : "tasks");
+    }
+
+    @Scheduled(fixedDelay = 10000)
     public void sendInformationEmail() {
-        long size = taskRepository.count();
         simpleEmailService.send(new Mail(
                 adminConfig.getAdminMail(),
                 SUBJECT,
-                "Currently in database you got: " + size + " tasks")
-        );
+                createMessage()));
```

Comment:
Nadal się nie rozumiemy w 100%. Napiszę, o co mi chodziło, a podczas spotkania to omówimy bardziej.
```
private String createMessage(long size) {
    return "Currently in database you got: " + size + (size == 1 ? "task" : "tasks");
}

public void sendInformationEmail() {
        simpleEmailService.send(new Mail(
                adminConfig.getAdminMail(),
                SUBJECT,
                createMessage(taskRepository.count())
        );
}
```

Reply:
Wada obecnego rozwiązania: dwa razy wołasz metodę `.count()` na repozytorium - pomyśl, że pod spodem może być (potencjalnie kosztowne) odpytanie bazy danych.